services/dynamic/dynamic-json.tester.js

```diff
@@ -57,8 +52,8 @@ t.create('JSON from url')
   )
   .expectJSON({
     name: 'custom badge',
-    value: 'gh-badges',
-    colorB: colorsB.brightgreen,
+    value: 'shields.io',
+    colorB: 'lightgreen',
```

Comment:
I think the previous value was right. This PR changes the default to 

![](https://shields-staging-pr-2399.herokuapp.com/badge/dynamic/json.svg?url=https%3A%2F%2Fraw.githubusercontent.com%2Fbadges%2Fshields%2Fmaster%2Fpackage.json&label=package&query=$.name) - https://shields-staging-pr-2399.herokuapp.com/badge/dynamic/json.svg?url=https%3A%2F%2Fraw.githubusercontent.com%2Fbadges%2Fshields%2Fmaster%2Fpackage.json&label=package&query=$.name

but I think we still want it to be

![](https://img.shields.io/badge/dynamic/json.svg?url=https%3A%2F%2Fraw.githubusercontent.com%2Fbadges%2Fshields%2Fmaster%2Fpackage.json&label=package&query=$.name) - https://img.shields.io/badge/dynamic/json.svg?url=https%3A%2F%2Fraw.githubusercontent.com%2Fbadges%2Fshields%2Fmaster%2Fpackage.json&label=package&query=$.name

Reply:
Oops! Thanks for catching that.